rmf_fleet_adapter/src/full_control/main.cpp

```diff
@@ -1234,18 +1234,28 @@ std::shared_ptr<Connections> make_fleet(
   const std::string finishing_request_string =
     node->declare_parameter("finishing_request", "nothing");
   rmf_task::ConstRequestFactoryPtr finishing_request = nullptr;
+  auto get_time =
+    []()
+    {
+      return std::chrono::steady_clock::now();
```

Comment:
We definitely do not want `steady_clock` here. That won't be a meaningful time stamp; it would just be the time since this node began running.

I would recommend using ROS2 time given by the ROS2 node's clock, or using `std::chrono::system_clock::now()` instead.

Reply:
Good point, I've made the change to use node time when it's available, otherwise `system_clock`, https://github.com/open-rmf/rmf_ros2/pull/267/commits/6815ea70e6fd9ec4fa03c82523c2c134c1aa9f50